### PR TITLE
#162342273 tests for article properties and relationships.

### PR DESCRIPTION
**Pull request by gbols**
#### What does this PR do?
It tests the article properties and its relationships.

#### Description of Task to be completed?
write tests to validate profile model properties and relationships.
#### How should this be manually tested?
none
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
#[162342273](https://www.pivotaltracker.com/story/show/162342273)
#### Screenshots (if appropriate)
none

### CODE REVIEW

**Comment by efalayi** [comment]
@gbols you have merge conflicts. Please kindly fix them.

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
+    "D": "^1.0.0",
```
Comment (efalayi) [inline package.json]: @gbols which dependency is `D`? Are we using it anywhere?

Comment (gbols) [reply package.json]: @efalayi sorry let me fix that

File: tests/models/article.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    })
+    it('defined a belongsTo association with UserModel', () => {
+      expect(Article.belongsTo.calledWith(User)).to.equal(true);
+      });
+
+    it('defined a belongsTo association with CategoryModel', () => {
+        expect(Article.belongsTo.calledWith(Category)).to.equal(true);
+      });
+  });
+});
```
Comment (efalayi) [inline tests/models/article.test.js]: @gbols a new line is required here.

**Review by efalayi — APPROVED** [review]
LGTM

**Pull request merged**